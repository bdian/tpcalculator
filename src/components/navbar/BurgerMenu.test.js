import React from 'react';
import BurgerMenu from './BurgerMenu';
import renderer from 'react-test-renderer';

test('Burger Menu snapshot', () => {
  const component = renderer.create(
    <BurgerMenu />,
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});