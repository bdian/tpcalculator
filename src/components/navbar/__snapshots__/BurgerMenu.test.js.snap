// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Burger Menu snapshot 1`] = `
<div
  className="sc-AxjAm kqgJvM"
>
  <div
    className=""
  >
    <span>
       
    </span>
    <span>
       
    </span>
    <span>
       
    </span>
  </div>
</div>
`;
